src/api/orderData.js

```diff
@@ -28,8 +28,22 @@ const createOrder = (cardObject) => new Promise((resolve, reject) => {
     }).catch(reject);
 });
 
+const getSingleOrder = (firebaseKey) => new Promise((resolve, reject) => {
+  axios.get(`${dbUrl}/orders/${firebaseKey}`.json)
```

Comment:
on this line, backtick should be on the other side of .json

Reply:
Nice catch, this is now added